### PR TITLE
Clean up Documentation for release v0.1.3

### PR DESCRIPTION
**Pull request by bouthilx**
Depends on #192 

### Add missing documentation for new features in v0.1.3

5563902 Add doc for PickledDB
7f188f9 Add doc for `test-db`
3fc9d08 Add missing working-dir and user to doc
5e4cf83 Add a note for mongodb for ssh tunnels

### CODE REVIEW

**Comment by bouthilx** [comment]
Tests should pass when #192 is merged. 

Compiled doc available at https://orion.readthedocs.io/en/feature-asha/index.html

File: docs/src/user/pytorch.rst [hunk 1]
```diff
@@ -168,6 +172,11 @@ Hunting Options
 
 The unique name of the experiment.
 
+``user``
+
+Username used to identify the experiments of a user. The default value is the systems username
```
Comment (DatCorno) [inline docs/src/user/pytorch.rst]: *system's*

File: docs/src/user/pytorch.rst [hunk 2]
```diff
@@ -181,6 +190,11 @@ The maximum number of trials tried during an experiment.
 
 The maximum number of trials to be executed by a worker (a single call to ``orion hunt [...]``).
 
+``working-dir``
+
+The directory where temporary configuration files are created. If not specified, Oríon will create a
```
Comment (DatCorno) [inline docs/src/user/pytorch.rst]: They're not temporary if `--working-dir` is set though.

Comment (bouthilx) [reply docs/src/user/pytorch.rst]: Indeed!

**Pull request merged**